pwnlib/tubes/ssh.py

```diff
@@ -942,7 +950,7 @@ def func(): pass
 except NameError:
     integer_types = int,
 exe   = %(executable)r
-argv  = [bytes(a) for a in %(argv)r]
+argv  = %(argv)r
```

Comment:
I wrote that here since the os.exec* functions complain about bytearray not being bytes. So it was the safest I guess.

If we really want to support everything everything, then we would have to convert all the things to some hex sequences or using pickle. And to deserialize it in some super-python1-compliant way in the best case and the bestestest would be to support deserializing and launching the target executable using perl, awk, and other weird utilities. But I really think that anything beyond py2.6 is at least as dead as [QMAGIC](https://github.com/torvalds/linux/blob/master/include/uapi/linux/a.out.h).